plugins/modules/inventory_intent.py

```diff
@@ -190,7 +190,9 @@
         type: bool
         default: False
       credential_update:
-        description: Make this as true needed for the updation of device credentials and other device details.
+        description: Make this as true needed for the updation of device credentials and other device details. Also if this parameter
```

Comment:
What happens if the device is not presented? Can you please write that as well?

Reply:
Update the docstring for the result if device is not present.